### PR TITLE
[JBRULES-3163] Minor modifications to make the compiler run with java7

### PR DESCRIPTION
**Pull request by anpieber**
I'm not sure if those are all, but at least for me it works after them

### CODE REVIEW

**Comment by etirelli** [comment]
Thanks. Applied.

File: drools-compiler/src/main/java/org/drools/commons/jci/compilers/EclipseJavaCompilerSettings.java [hunk 1]
```diff
@@ -85,6 +85,7 @@ public final class EclipseJavaCompilerSettings extends JavaCompilerSettings {
         put("1.4", CompilerOptions_VERSION_1_4);
         put("1.5", CompilerOptions_VERSION_1_5);
         put("1.6", CompilerOptions_VERSION_1_6);
+        put("1.7", CompilerOptions_VERSION_1_6);
```
Comment (anpieber) [inline drools-compiler/src/main/java/org/drools/commons/jci/compilers/EclipseJavaCompilerSettings.java:88]: woha. one thing I just seen... Is there a CompilerOptions_VERSION_1_7?

Comment (etirelli) [inline drools-compiler/src/main/java/org/drools/commons/jci/compilers/EclipseJavaCompilerSettings.java:88]: yeah, fixed that. Thanks.

Comment (anpieber) [inline drools-compiler/src/main/java/org/drools/commons/jci/compilers/EclipseJavaCompilerSettings.java:88]: Thanks for fixing it and sorry for the error in the pull-request :-( Stupid
copy-paste-modify error...

**Pull request closed without merging**